src/nrc/conf/includes/custom_notifications.py

```diff
@@ -0,0 +1,65 @@
+from django_setup_configuration.configuration import BaseConfigurationStep
+from django_setup_configuration.models import ConfigurationModel, DjangoModelRef
+from notifications_api_common.contrib.setup_configuration.models import (
+    NotificationConfigurationModel,
+)
+from notifications_api_common.contrib.setup_configuration.steps import (
+    get_service,
+)
+from notifications_api_common.models import NotificationsConfig
+from pydantic import Field
+
+
+class CustomNotificationConfigurationModel(ConfigurationModel):
+    notifications_api_service_identifier: str = Field(
+        default="",
+        metadata={
+            "django_model_ref": DjangoModelRef(
+                NotificationsConfig,
+                "notifications_api_service",
+                examples=["notificaties-api"],
+                null=True,
+            )
+        },
+    )
+
+    class Meta:
+        django_model_refs = {
+            NotificationsConfig: [
+                "notification_delivery_max_retries",
+                "notification_delivery_retry_backoff",
+                "notification_delivery_retry_backoff_max",
+            ]
+        }
```

Comment:
I think it would be nice if this could inherit from the model in `notifications_api_common` and just override the notifications_api_service_identifier

```suggestion
from notifications_api_common.contrib.setup_configuration.models import NotificationConfigurationModel as _NotificationConfigurationModel

class NotificationConfigurationModel(_NotificationConfigurationModel):
    # add whatever changes are needed to make it not required
    notifications_api_service_identifier: str = DjangoModelRef(
        NotificationsConfig,
        "notifications_api_service",
        examples=["notificaties-api"],
    )
```

and then for the step, you only can also inherit from the existing step in `notifications_api_common` and only override the `config_model` to use the model above